src/pages/FormNewDay.js

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useState } from 'react';
+import x_icon from '../img/icon_x.svg';
+import FormDay from '../components/FormDay.js';
+import Modal from '../components/Modal.js';
+
+export default function FormNewDay({
+  onEditDestination,
+  onGetCurrentDestination,
+}) {
+  const { id } = useParams();
+  const [isOpen, setIsOpen] = useState(false);
+  const navigate = useNavigate();
+  const [currentDestination, setCurrentDestination] = useState(
+    onGetCurrentDestination(id)
+  );
+
+  function showSubmitMessage() {
+    setIsOpen(true);
+    setTimeout(() => {
+      setIsOpen(false);
+      navigate(-1);
+    }, 2500);
+  }
+
+  const onSubmit = (data, formLocations) => {
+    const routes = {
+      date: data.date,
+      locations: formLocations
+        .filter(location => location.isChecked === true)
+        .map(location => location.location),
+    };
+    onEditDestination(currentDestination[0], routes);
+    showSubmitMessage();
+  };
+
```

Comment:
Nice work  with the setTimeout ;) Is there an error message sometimes according to it? if yes you could also place it in a useeffect! I´ve got the problem with it an placed it into a useeffect

Reply:
actually not no, I haven't got any problems with modals at all yet...